phy/cluster/manual/session.py

```diff
@@ -201,18 +188,36 @@ def _store_cluster(self,
                               append=True,
                               )
 
-    def store_all_clusters(self, spikes_per_cluster):
+    def is_consistent(self, cluster, spikes):
+        """Return whether the data of a cluster is consistent."""
```

Comment:
In fairness this is probably more "Return whether the filesize of a cluster store file is correct" - you're not checking the data at all, even less for consistency!